content/styles/00-base/_body.css

```diff
@@ -0,0 +1,28 @@
+@charset "UTF-8";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%; /* 10px */
+  font-size: calc(1em * 0.625); /* fix */
+}
+
+body {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  font-size: 1.7rem;
+  line-height: 1.5;
+  margin: 0;
+  color: var(--black);
+  font-weight: 400;
+  font-family: var(--content-font), var(--head-font), Arial, Helvetica, sans-serif;
```

Comment:
Wondering if we really want the head font to be used in the content as fallback.
How did we end up with a sans-serif font for both titles and contents BTW?

Reply:
good point